modules/solid_mechanics/test/tests/j_integral/tests

```diff
@@ -58,18 +58,18 @@
  [../]
  [./j_3d_as_2d_topo_q_outq]
    type = 'Exodiff'
-   input = 'j_integral_3d_as_2d_topo_q_func.i cli_args = "DomainIntegral/output_q=false" "Outputs/file_base=j_integral_3d_as_2d_topo_q_func_outputQ_out"'
+   input = 'j_integral_3d_as_2d_topo_q_func.i "DomainIntegral/output_q=false" "Outputs/file_base=j_integral_3d_as_2d_topo_q_func_outputQ_out"'
```

Comment:
This does not look like the right fix

Reply:
What do you think it should be?  It passes.

Reply:
Maybe

```
input = j_integral_3d_as_2d_topo_q_func.i 
cli_args = 'DomainIntegral/output_q=false Outputs/file_base=j_integral_3d_as_2d_topo_q_func_outputQ_out'
```

Tacking the CLI args onto the input file name parameter seems rather ugly.